app/code/VConnect/OrderVolume/Model/Product/Config.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace VConnect\OrderVolume\Model\Product;
+
+class Config
+{
+    public const PRODUCT_VOLUME = 'volume';
```

Comment:
please declare it as a public constant in your patch and call it from there. Don't need to create separate class for this